apps/frontend/core-dapp/e2e/utils/tenderly.ts

```diff
@@ -45,3 +45,25 @@ export async function addBalanceToWallet({
 }): Promise<void> {
   await provider.send('tenderly_addBalance', [[address], '0x21e19e0c9bab2400000'])
 }
+
+export async function overrideState({
+  contract,
+  slot,
+  value,
+  provider,
+}: {
+  contract: string
+  slot: string
+  value: Buffer
+  provider: ethers.providers.JsonRpcProvider
+}): Promise<void> {
+  if (value.byteLength !== 32) throw new Error('Invalid state value length')
+  await provider.send('tenderly_setStorageAt', [contract, slot, `0x${value.toString('hex')}`])
+}
+
+export function bnToBytesArray32(bn: ethers.BigNumber): Buffer {
+  const bnBytes = Buffer.from(bn.toHexString().replace('0x', ''), 'hex')
+  const target = Buffer.alloc(32, 0)
+  bnBytes.copy(target, target.length - bnBytes.length)
+  return target
+}
```

Comment:
I'd recommend using AbiCoder instead of writing our own code to encode value

With AbiCoder:

```suggestion
const abiCoder = new AbiCoder()

export async function overrideState({
  contract,
  slot,
  value,
  provider,
}: {
  contract: string
  slot: string
  value: BigNumber
  provider: ethers.providers.JsonRpcProvider
}): Promise<void> {
  const encodedValue = abiCoder.encode(['uint'], [value])
  await provider.send('tenderly_setStorageAt', [contract, slot, encodedValue])
}
```